package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "uk.co.compucorp.civicase",
+  "version": "2.0.0",
```

Comment:
This should be `"5.0"`. And when we release civicase, we will change it to `"5.1"`

Reply:
Done!